### PR TITLE
Removed TheSchrooms and added ClayMC.net Network

### PR DESCRIPTION
**Pull request by TVGolgolex**
Removed TheSchrooms
Added ClayMC.net Network

## Type of change

- [x] Add new directory for a server
- [x] Update directory of a server
- [x] Remove directory of a server
- [ ]  Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/claymcnetwork/background.png
Comment (DoJapHD) [inline minecraft_servers/claymcnetwork/background.png:1]: Compress

File: minecraft_servers/claymcnetwork/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/claymcnetwork/background@2x.png:1]: Compress

File: minecraft_servers/claymcnetwork/manifest.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "server_name": "claymc",
+  "nice_name": "ClayMC",
+  "direct_ip": "play.claymc.net",
+  "server_wildcards": [
+    "%.claymc.net"
```
Comment (DoJapHD) [inline minecraft_servers/claymcnetwork/manifest.json]: Not a real wildcard
https://i.imgur.com/pFAE7RA.png

File: minecraft_servers/claymcnetwork/manifest.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  ],
+  "supported_languages": [
+    "de", "en"
+  ],
+  "social": {
+    "web": "https://claymc.net",
+    "web_shop": "https://shop.claymc.net",
```
Comment (DoJapHD) [inline minecraft_servers/claymcnetwork/manifest.json]: Not reachable

File: minecraft_servers/claymcnetwork/manifest.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    "twitter": "ClayMCNetwork",
+    "discord": "https://discord.claymc.net",
```
Comment (DoJapHD) [inline minecraft_servers/claymcnetwork/manifest.json]: Custom links are reserved for partners; use https://discord.gg/[INVITE]

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by TVGolgolex** [comment]
Check failed although there is no "/" at the end of the link.

**Comment by DoJapHD** [comment]
Remove the whole "theshrooms" folder in your branch aswell, please

**Comment by TVGolgolex** [comment]
Now something has gone wrong. I'll push it again.

File: minecraft_servers/claymc/manifest.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
+{
+  "server_name": "claymc",
+  "nice_name": "ClayMC",
+  "direct_ip": "play.claymc.net",
+  "server_wildcards": [
+    "%.claymc.net"
```
Comment (maaaathis) [inline minecraft_servers/claymc/manifest.json:6]: Doesn't look like a real wildcard
image

File: minecraft_servers/claymc/manifest.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  ],
+  "supported_languages": [
+    "de", "en"
+  ],
+  "social": {
+    "web": "https://www.claymc.net",
```
Comment (maaaathis) [inline minecraft_servers/claymc/manifest.json:12]: Not reachable

File: minecraft_servers/claymc/manifest.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
+{
+  "server_name": "claymc",
+  "nice_name": "ClayMC",
+  "direct_ip": "play.claymc.net",
+  "server_wildcards": [
+    "%.claymc.net"
+  ],
+  "supported_languages": [
+    "de", "en"
```
Comment (maaaathis) [inline minecraft_servers/claymc/manifest.json:9]:
```suggestion
    "de",
    "en"
```

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/claymc/manifest.json (hunk 4 above)
Comment (TVGolgolex) [reply minecraft_servers/claymc/manifest.json:6]: As I already said. There's a A-Record on the ClayMC.net Domain

Comment (maaaathis) [reply minecraft_servers/claymc/manifest.json:6]: But thats not a **wilcard** record: https://en.wikipedia.org/wiki/Wildcard_DNS_record

Comment (TVGolgolex) [reply minecraft_servers/claymc/manifest.json:6]: Fixed

Comment (maaaathis) [reply minecraft_servers/claymc/manifest.json:6]: Looks good! 👍
image

**Comment by DoJapHD** [comment]
Is the server even online?
https://i.imgur.com/XPlxMGv.png

**Comment by TVGolgolex** [comment]
Server is online but there's a network issue with the hosting

**Comment by DoJapHD** [comment]
Server is still offline;
Reopen PR once server is available

**Pull request closed without merging**